### ISSUE TITLE
Openbabel hydrogenate_and_compute_partial_charges fails on some machines

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#105]
```
rbharath@not0rious:~/deepchem$ nosetests -v deepchem.featurizers.tests.test_nnscore --nocapture
Run simple tests with NNScore. ...
Illegal instruction (core dumped)
```

This seems to happen on certain older machines. It's possible that openbabel versions from `omnia` are built with new glibc or something of the like. I'm not sure if there's any easy fix for this issue except avoidance.

**Comment by evanfeinberg** [comment]
Does this seem to occur with a certain type of chemical system or does it
seem more dependent on the hardware? I haven't seen
hydrogenate_and_compute_partial_charges fail yet with the Dud-E ligands,
for example.

On Sun, Feb 21, 2016 at 7:15 PM, Bharath Ramsundar <notifications@github.com

**Comment by rbharath** [comment]
I think this is a hardware/system issue. If openbabel runs at all, it seems to be pretty stable across chemical systems.

**Comment by hainm** [comment]
might be due to omnia build on new system. For example I can not use pymol from omnia channel since my computer has old gcc.

```bash
ImportError: /lib64/libc.so.6: version `GLIBC_2.7' not found
```

**Comment by rbharath** [comment]
I think this issue might now be fixed due to the upgrade to Openbabel 2.4 (at least one user reported no longer seeing this problem with latest openbabel). Could anyone else seeing this problem give it a try to see if it works for them? (Updated install directions now on `README.md` for github)

**Comment by rbharath** [comment]
Going to close. Can reopen if we see this issue again.
